rel/overlay/etc/default.ini

```diff
@@ -108,6 +108,9 @@ bind_address = 127.0.0.1
 ; Maximum allowed http request size. Applies to both clustered and local port.
 ;max_http_request_size = 4294967296 ; 4GB
 
+; Decode + to space. Does not apply to dbname.
```

Comment:
It may be worth it to explain that it will decode the dbname into a space, it's just that spaces are not allowed for dbnames so there won't be a compatibility issue when toggling it between true and false.

Reply:
I have changed it to `Set to true to decode + to space in db and doc_id parts.`